engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -168,6 +168,11 @@ public Boolean get() {
     @Override
     public void update(float delta) {
         if (metricsLabel != null) {
+            if (metricsModes.isEmpty()) {
+                metricsLabel.setText("No metric modes are available!");
+                return;
+            }
+
             metricsLabel.setText(metricsModes.get(currentMode).getMetrics());
```

Comment:
I think the typical case is that the metricsLabel is not empty. So, let's put that in the first part of the "if" block and let's handle the metricsMode.isEmpty() case with an "else" block.

Also, are there situations in which the metricsLabel is null?